packages/ibm-products/src/components/Checklist/Checklist.js

```diff
@@ -0,0 +1,336 @@
+      >
+        {(title || chartLabelAndValue) && (
+          <header className={`${blockClass}__header`}>
+            {chartLabelAndValue && (
+              <ChecklistChart
+                aria-labelledby={chartLabelId}
+                theme={theme}
+                value={chartValue}
+              />
+            )}
+
+            <div className={`${blockClass}__titles`}>
+              {title && <h2 className={`${blockClass}__title`}>{title}</h2>}
+
+              {chartLabelAndValue && (
+                <h3 id={chartLabelId} className={`${blockClass}__chart-label`}>
+                  {chartLabel}
+                </h3>
+              )}
+            </div>
+
+            {showToggle && (
+              <Button
+                aria-controls={listContainerId}
+                aria-expanded={isOpen}
+                aria-label={checklistToggleAriaLabel}
+                className={`${blockClass}__toggle`}
+                kind="ghost"
+                onClick={handleClickToggle}
+                size="sm"
+              >
+                <ChevronUp size={16} className={cx(`${blockClass}__chevron`)} />
+              </Button>
+            )}
+          </header>
+        )}
+
+        <div id={listContainerId} className={`${blockClass}__content-outer`}>
+          <div className={`${blockClass}__content-inner`}>
+            <section className={cx(`${blockClass}__body`)}>
+              {taskLists.map((list, index) => {
+                return (
+                  <div
+                    className={`${blockClass}__list-group`}
+                    key={`${list.title}-${index}`}
+                  >
+                    {list.title && (
+                      <h3
+                        title={list.title}
+                        className={`${blockClass}__list-title`}
+                      >
+                        {list.title}
+                      </h3>
+                    )}
+                    <ol className={`${blockClass}__list`}>
+                      {list.tasks.map((item, index) => {
+                        return (
+                          <li
+                            className={`${blockClass}__list-item`}
+                            key={`${item.label}-${index}`}
+                          >
+                            <ChecklistIcon kind={item.kind} />
+
+                            {typeof item.onClick === 'function' ? (
+                              <Button
+                                className={cx(`${blockClass}__button`, {
+                                  [`${blockClass}__button--error`]:
+                                    item.kind === 'error',
+                                })}
+                                onClick={() => {
+                                  item.onClick(item);
+                                }}
+                                size="sm"
+                              >
+                                <div title={item.label}>{item.label}</div>
```

Comment:
Do you need the extra `div` inside of this button? I think you should be able to set the `tittle` attribute on the button.